relay-general/src/store/mod.rs

```diff
@@ -53,6 +54,8 @@ pub struct StoreConfig {
 
     /// Emit breakdowns based on given configuration.
     pub breakdowns: Option<normalize::breakdowns::BreakdownsConfig>,
+
+    pub span_attributes: BTreeSet<String>,
```

Comment:
@Zylphrex it might be better to wrap this in `Option<T>`. In other words, `Option<BTreeSet<String>>`.

I think project config is not guaranteed to contain `span_attributes`.

Reply:
I don't think that's necessary since `serde` will use the default value during deserialization if necessary. In this case, the default value is `BTreeSet::new()`.